lib/time-uuid.js

```diff
@@ -99,10 +112,10 @@ TimeUuid.max = function (date, ticks) {
  * If not provided a random clockId will be generated.
  */
 TimeUuid.now = function (nodeId, clockId) {
-    return new TimeUuid(null, null, nodeId, clockId);
+  return new TimeUuid(null, null, nodeId, clockId);
 };
 
-var uuidRe = /^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$/;
+let uuidRe = /^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$/;
```

Comment:
It's never reassigned, so it's a `const`. Try using `const` as much as possible, immutability is our friend :)